### ISSUE TITLE
Section4_Sprint1

### ISSUE THREAD

**Issue by jangsik-park** [jangsik-park/Study_list#1]
# N411a

- **신경망을 이용하여 이진분류 문제 해결하기**
- **optimizer , loss, metrics, epochs, batch-size**

  `model.predict_classes(X_test).flatten() ## class 예측값 구하는 방법`

- **Optimizer에 대한 조사하기 _(도전과제)_**
  - 정의
  - 종류

# N413

- L1/L2 regularization 간단 정리

- 학습 규제 전략 = Regularization Strategies
  - Overfitting 극복을 위한 노력
  - Early Stopping
  - weight decay
  - weight Constraint
  - Drop Out 사용

-  Keras에서 규제 방법 구현

- 더 나은 모델 학습을 위한 방법
  - Learning rate decay
  - Learning rate 스케쥴링

- Activation Fuction 추가 종류
  - Tann Function
  - Leaky ReLU
  - Softmax Function
